canu/2.0/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Use ubuntu as base image
+FROM ubuntu:xenial
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
+LABEL software="Canu"
+LABEL software.version="2.0; 2.17; 1.4.3"
+LABEL description="Long read assembly and polishing tools"
+LABEL website="https://github.com/marbl/canu"
+LABEL license="See program repositories"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
+# install python
+RUN apt-get update && apt-get install -y \
+   perl \
+   default-jre \
+   gnuplot \
+   libgcc-5-dev \
+   libgomp1 \
+   maven \
+   git \
+   wget \
+   python3 \
+   build-essential \
+   cmake \
+   zlib1g-dev && \
+   apt-get clean
+
+# install canu
+RUN wget https://github.com/marbl/canu/releases/download/v2.0/canu-2.0.Linux-amd64.tar.xz && \
+  tar -xvf canu-2.0.Linux-amd64.tar.xz && \
+  rm canu-2.0.Linux-amd64.tar.xz
+
+ENV PATH="/canu-2.0/Linux-amd64/bin:${PATH}"
+
+RUN mkdir /data
```

Comment:
could you combine this step with the RUN layer for downloading & installing canu?